openaq/openaq.org
# Update/node selection

- pulls data from averages endpoint and creates a single aggregate object for each parameter and location selected by the user
- splits the project dashboard into two options - initial project dashboard and dashboard with calculated averages
- adds a new timeseries table that doesn't make a call to the /averages endpoint since that is already done prior to rendering the chart -- this is slightly duplicative, but I thought it was most readable


Known Issues: some projects get 500 errors when selecting locations (ei 27581) I'm leaving an error message here instead of spending time tracking this down


To test:
UI - 
- select a project from the project hub, I like 27595 since it has multiple params to chose from
- in the map check 'select locations'
- click on locations and click 'select' add as many as you want, switch the params using the legend and add some from there
- select 'Apply Selection' and review the dashboard

Code: 
- I recommend reviewing the following to make sure they make sense and are readable
 -  `app/assets/scripts/views/project/nodes-dashboard.js`
 -  `app/assets/scripts/views/project/project.js`
 - `app/assets/scripts/components/dashboard/node-time-series-card.js`

this fully closes #483 with the exception of displaying all the locations as well as the averages on the timeseries card
